presto-parquet/src/main/java/com/facebook/presto/parquet/dictionary/FloatDictionary.java

```diff
@@ -31,7 +33,8 @@ public FloatDictionary(DictionaryPage dictionaryPage)
         super(dictionaryPage.getEncoding());
         content = new float[dictionaryPage.getDictionarySize()];
         FloatPlainValuesReader floatReader = new FloatPlainValuesReader();
-        floatReader.initFromPage(dictionaryPage.getDictionarySize(), dictionaryPage.getSlice().getBytes(), 0);
+        final ByteBufferInputStream in = ByteBufferInputStream.wrap(ByteBuffer.wrap(dictionaryPage.getSlice().getBytes()));
```

Comment:
ditto about naming(`in` -> `inputStream`) and `final` modifer

Reply:
Done.